### ISSUE TITLE
ndef gui paramview backwards compatibility

### ISSUE THREAD

**Issue by vividsnow** [supercollider/supercollider#2158]
Hi

I updated to the master branch and got unexpected behavior of NdefGui views: 2-elements array params are forced to display as rangers.

Previously it was like:
https://github.com/supercollider/supercollider/blob/3.7/SCClassLibrary/JITLib/GUI/EnvirGui.sc#L410

and now, two conditions are dropped ("useRanger" and "key.asSpec.isNil"):
https://github.com/supercollider/supercollider/blob/master/SCClassLibrary/JITLib/GUI/ParamView.sc#L68

**Comment by adcxyz** [comment]
closed with 9e9d78b.

tests: 
```
m = NdefMixer(s, 5);
m.switchSize(2);

Ndef(\testy, { SinOsc.ar(\freq.kr(400)) * 0.2 });
m.editGui.object_(Ndef(\testy)); 
// -> slider for freq param

// switch to a param with an array of two values
Ndef(\testy2, { SinOsc.ar(\freq.kr([300, 400])) * 0.2 });
m.editGui.object_(Ndef(\testy2)); 
// -> displays as ranger now

// turn useRanger off
m.editGui.paramGui.useRanger = false;
Ndef(\testy2).set(\freq, [345, 456]); // switches when new value is set
m.editGui.paramGui.useRanger = true;
Ndef(\testy2).set(\freq, [456, 789]); 
```

**Comment by vividsnow** [comment]
thanks!
